src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzer.cs

```diff
@@ -693,10 +740,50 @@ private static ISymbol GetEventAccessor(IEventSymbol iEvent, IOperation operatio
             }
             else
             {
-                CheckOperationAttributes(operation, context, platformSpecificOperations, platformSpecificMembers, msBuildPlatforms, symbol);
+                if (!CheckOperationAttributes(operation, context, platformSpecificOperations, platformSpecificMembers, msBuildPlatforms, symbol) && symbol is IMethodSymbol method)
```

Comment:
Here we are checking the operation (method) first, if it has attributed, else we will check the method's type parameters

Reply:
Ah got you, thanks. Can we add unit test that I mentioned in https://github.com/dotnet/roslyn-analyzers/pull/4403#discussion_r517676488?

Reply:
Sure i will add a test for that